projects/packages/connection-ui/_inc/components/admin/style.scss

```diff
@@ -0,0 +1,18 @@
+#jetpack-connection-ui-container {
+  max-width: 800px;
```

Comment:
I'd suggest using something like Prettier to make sure that all your spaces are converted into tabs on save, so we remain consistent with the rest of the codebase.

Reply:
Fixed now, thanks!